docs/modes-of-operation.rst

```diff
@@ -140,6 +140,12 @@ Slave operation can also be programmed using several
 command is especially useful as it triggers an immediate retrieval of
 the zone from the configured master.
 
+Zone transfers are added to a queue and processed according to priority
```

Comment:
The documentation covers more than the current version, so perhaps something like:
```suggestion
Since 4.5.0, zone transfers are added to a queue and processed according to priority
```